app/uk/gov/hmrc/agentauthorisation/connectors/InvitationsConnector.scala

```diff
@@ -67,12 +67,17 @@ class InvitationsConnector @Inject()(
   private[connectors] def cancelInvitationUrl(arn: Arn, invitationId: InvitationId) =
     new URL(baseUrl, s"/agent-client-authorisation/agencies/${arn.value}/invitations/sent/${invitationId.value}/cancel")
 
-  private[connectors] def getAgencyInvitationsUrl(arn: Arn, createdOnOrAfter: LocalDate): URL =
+  private[connectors] def getAgencyInvitationsUrl(
+    arn: Arn,
+    supportedServices: Seq[String],
+    createdOnOrAfter: LocalDate): URL = {
+    val services = s"service=${supportedServices.mkString(",")}&"
```

Comment:
this construction each time the above method is called , instead simply declare 
`val services = "HMRC-MTD-IT,HMRC-MTD-VAT`  and use in the URL straightaway